doc/classes/ConfigFile.xml

```diff
@@ -84,6 +84,45 @@
 		}
 		[/csharp]
 		[/codeblocks]
+		If a game's difficulty is enumerated, the example below shows how the game's configuration file could store a comment as a guide to each value:
+		[codeblocks]
+		[gdscript]
+		enum Difficulty {
+		    CASUAL = 0,
+		    NORMAL = 1,
+		    HARD = 2,
+		    SAFETY = 3,
+		}
+
+		var player_difficulty = Difficulty.NORMAL
+
+		func create_config():
+		    var config = ConfigFile.new()
+		    config.set_value("game", "difficulty", player_difficulty)
+		    config.set_section_key_comment("game", "difficulty", "0: Casual\n1: Normal\n2: Hard\n3: Safety")
```

Comment:
This may be also pushing the line length, by the way. It won't look exactly pretty in the built-in docs.
```suggestion
		    config.set_section_key_comment("game", "difficulty", "0: Easy\n1: Normal\n2: Hard")
```